localtunnel.js

```diff
@@ -1,9 +1,9 @@
 const localtunnel = require('localtunnel')
 console.log(`Start tunneling localhost: ${process.env.PORT} at https://${process.env.LOCALSUBDOMAIN}.localtunnel.me`)
-localtunnel(process.env.PORT, { subdomain: process.env.LOCALSUBDOMAIN }, (err, tunnel) => {
+localtunnel(process.env.PORT, { subdomain: process.env.LOCALSUBDOMAIN }, err => {
     if (err){
         console.log('Err: ', err)
     } else {
-        console.log("Tunneling goingOn ", process.env.PORT)
+        console.log("Tunneling running on port ", process.env.PORT)
     }
 })
```

Comment:
@b-dabrowski said he had some problems with this script, maybe we should add some event listeners on failures/disconnects?

https://github.com/localtunnel/localtunnel#tunnel